lib/veteran_status_card/service.rb

```diff
@@ -0,0 +1,415 @@
+
+      return VeteranStatusCard::Constants::EDIPI_NO_PNL_RESPONSE if ssc_code == EDIPI_NO_PNL_CODE
+
+      return VeteranStatusCard::Constants::CURRENTLY_SERVING_RESPONSE if CURRENTLY_SERVING_CODES.include?(ssc_code)
+
+      return VeteranStatusCard::Constants::ERROR_RESPONSE if ERROR_SSC_CODES.include?(ssc_code)
+
+      # Default fallback
+      VeteranStatusCard::Constants::ERROR_RESPONSE
+    end
+
+    ##
+    # Returns the user's normalized full name components
+    #
+    # @return [Hash] the user's full name in normalized format with keys :first, :middle, :last, and :suffix
+    #
+    def full_name
+      @user.full_name_normalized
+    end
+
+    ##
+    # Gets the user's combined disability rating percentage
+    # Uses Lighthouse API if enabled, otherwise falls back to EVSS
+    # Returns nil if both services fail
+    #
+    # @return [Integer, nil] the combined disability rating percentage or nil on error
+    #
+    def disability_rating
+      lighthouse? ? lighthouse_rating : evss_rating
+    rescue => e
+      Rails.logger.error("Disability rating error: #{e.message}", backtrace: e.backtrace)
+      nil
+    end
+
+    ##
+    # Checks if Lighthouse rating API is enabled for this user
+    #
+    # @return [Boolean] true if Lighthouse is enabled, false otherwise
+    #
+    def lighthouse?
+      Flipper.enabled?(:profile_lighthouse_rating_info, @user)
+    end
+
+    ##
+    # Gets the disability rating from Lighthouse API
+    # Returns nil if service call fails or user missing ICN
+    #
+    # @return [Integer, nil] the combined disability rating percentage from Lighthouse or nil on error
+    #
+    def lighthouse_rating
+      return nil if @user.icn.blank?
+
+      lighthouse_disabilities_provider.get_combined_disability_rating
+    rescue => e
+      Rails.logger.error("Lighthouse disabilities error: #{e.message}", backtrace: e.backtrace)
+      nil
+    end
+
+    ##
+    # Returns the Lighthouse disabilities provider instance (memoized)
+    #
+    # @return [LighthouseRatedDisabilitiesProvider] the provider instance
+    #
+    def lighthouse_disabilities_provider
+      @lighthouse_disabilities_provider ||= LighthouseRatedDisabilitiesProvider.new(@user.icn)
+    end
+
+    ##
+    # Gets the disability rating from EVSS API
+    # Returns nil if service call fails
+    #
+    # @return [Integer, nil] the combined disability rating percentage from EVSS or nil on error
+    #
+    def evss_rating
+      return nil if auth_headers.nil?
+
+      evss_service.get_rating_info
+    rescue => e
+      Rails.logger.error("EVSS rating error: #{e.message}", backtrace: e.backtrace)
+      nil
+    end
+
+    ##
+    # Returns the EVSS service instance (memoized)
+    #
+    # @return [EVSS::CommonService] the EVSS service instance
+    #
+    def evss_service
+      @evss_service ||= EVSS::CommonService.new(auth_headers)
```

Comment:
‼️ The external service EVSS has been decommissioned. Can you refactor this file so it's not checking EVSS? (logging messages shouldn't include "EVSS" either).